sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4511,6 +4535,46 @@ public partial class ManagedDiskPatch
         public static bool operator !=(Azure.ResourceManager.Compute.Models.NetworkApiVersion left, Azure.ResourceManager.Compute.Models.NetworkApiVersion right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct NetworkInterfaceAuxiliaryMode : System.IEquatable<Azure.ResourceManager.Compute.Models.NetworkInterfaceAuxiliaryMode>
```

Comment:
We need to add a prefix to this type because the network RP already has this type.
Could `VirtualMachine` work? I notice it is also used in VMSS related models.

Reply:
addressed

Reply:
could we keep it the same name as the networkRP's ? it is referring to the same thing. 
They NetworkInterfaceAuxiliaryMode has 1 more enum value in networkRP https://github.com/Azure/azure-rest-api-specs/blob/eaaa0d77e4033ebf60e7dd77eec56f4160b95cf4/specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/networkInterface.json#L1222
and NetworkInterfaceAuxiliarySku enums are exactly the same enums in both compute and network. https://github.com/Azure/azure-rest-api-specs/blob/eaaa0d77e4033ebf60e7dd77eec56f4160b95cf4/specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/networkInterface.json#L1237

Reply:
@ArcturusZhang

Reply:
renamed them to ComputeNetworkInterfaceAuxiliaryMode and ComputeNetworkInterfaceAuxiliarySku since they exist for both VM and VMSS